apiary.apib

```diff
@@ -308,7 +326,25 @@ Get details for a single department.
         + data  (object)
             + id: `67c576e7-7e6f-465d-b6ab-a864f6e5e95b` (string)
             + name: `Human Resources` (string)
+            + vat_number: `BE0899623035` (string)
+            + address (Address)
+            + emails (array)
+                + (object)
+                    + type: `primary` (enum[string])
+                        + Members
+                            + primary
+                            + invoicing
+                    + email: `info@piedpiper.eu` (string)
+            + telephones (array[object])
+                + type: `phone` (enum[string], required)
+                    + Members
+                        + phone
+                        + fax
+                + number: `092980615` (string, required)
+            + website: `https://piedpiper.com` (string)
             + currency: `EUR` (string)
+            + iban: `BE12123412341234` (string)
```

Comment:
B/Q: why are we adding this to the root level? This for me seems to be missing a "bank information" object and then having these fields inside that object.

Reply:
Lots of endpoints are adding IBAN and BIC on the top level right now. This might be a BC breaking improvement that we can release in a next API version.